module/BlockSelector.h

```diff
@@ -38,33 +38,9 @@ class BlockSelector {
   std::array<bool, static_cast<int>(CIRCLE_ID::ID7) + 1> arrivableCircles;
 
   // ブロックがなくなった際、到着可能になるサークル
-  const std::array<std::array<bool, static_cast<int>(CIRCLE_ID::ID7) + 1>,
-                   static_cast<int>(BLOCK_ID::ID7) + 1>
-      // ブロックが運搬された際に開放されるブロックサークル
-      OPEN_CIRCLE_ID = { {
-          { T, T, T, T, F, F, F, F },
-          { F, T, T, F, T, F, F, F },
-          { T, T, F, T, F, T, F, F },
-          { T, T, T, T, T, F, T, T },
-          { T, T, F, T, T, T, T, T },
-          { F, F, T, F, T, F, T, T },
-          { F, F, F, T, F, T, T, F },
-          { F, F, F, F, T, T, T, T },
-      } };
+  const std::array<std::array<bool, static_cast<int>(BLOCK_ID::ID7) + 1>, 8> OPEN_CIRCLE_ID;
```

Comment:
```suggestion
  const std::array<std::array<bool, static_cast<int>(BLOCK_ID::ID7) + 1>, static_cast<int>(CIRCLE_ID::ID7) + 1> OPEN_CIRCLE_ID;
```
Approveしてしまったけど、配列の要素数をマジックナンバーに変更したのはなんで？

Reply:
キャストしてるのがいけないのかなって思ってマジックナンバーで試したまま，直すの忘れてた
直します